modules/eco-about-node-maintenance-standalone.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+// nodes/nodes/eco-node-maintenance-operator.adoc
+
+:_content-type: CONCEPT
+[id="eco-about-node-maintenance-operator_{context}"]
+= About the Node Maintenance Operator
+
+You can place nodes into maintenance mode using the `oc adm` utility, or using `NodeMaintenance` custom resources (CRs).
+
+Placing a node into maintenance marks the node as unschedulable and drains all the virtual machines and pods from it. Virtual machine instances that have a `LiveMigrate` eviction strategy are live migrated to another node without loss of service. This eviction strategy is configured by default in virtual machine created from common templates but must be configured manually for custom virtual machines.
```

Comment:
Virtual machines are not interesting for standalone NMO, because you should not have both at the same time for 4.10. Since we keep the Virtulaization - NMO docs, I would remove all references to VMs here.
For 4.11 we can consider where to place VM specific docs, here or at Virt docs.

Reply:
that only leaves `Placing a node into maintenance marks the node as unschedulable and drains all pods from it.` at this place, and then it continues at line 19.